tests/test_project.py

```diff
@@ -8,3 +10,27 @@ def test_project_folder(cookies):
     assert project.exception is None
     assert project.project.basename == 'my-python-project'
     assert project.project.isdir()
+
+
+def run(command: str, dirpath: os.PathLike) -> subprocess.CompletedProcess:
+    return subprocess.run(shlex.split(command),
+                          stdout=subprocess.PIPE,
+                          stderr=subprocess.PIPE,
+                          cwd=dirpath,
+                          text=True)
+
+
+def test_pytest(cookies):
+    result = cookies.bake()
+    env_output = run('python3 -m venv env', result.project)
+    assert env_output.returncode == 0
+    latest_pip_output = run('env/bin/pip3 install --upgrade pip setuptools', result.project)
```

Comment:
Should we also add `wheel`?

```suggestion
    latest_pip_output = run('env/bin/pip3 install --upgrade pip wheel setuptools', result.project)
```

Reply:
That would be for a test which builds distros, we can add it there.

Reply:
ok